src/providers/symbolProvider.ts

```diff
@@ -38,6 +38,20 @@ export class SymbolProvider implements vscode.DocumentSymbolProvider {
             );
         }
 
+        for (const hotkey of script.hotkeys) {
+            result.push(
+                new vscode.SymbolInformation(
+                    hotkey.name,
+                    vscode.SymbolKind.Event,
+                    null,
+                    new vscode.Location(
+                        hotkey.document.uri,
+                        new vscode.Position(hotkey.line, hotkey.character),
+                    ),
+                ),
+            );
+        }
```

Comment:
```suggestion
        result.push(...script.hotkeys.map(hotkey => (
            new vscode.SymbolInformation(
                hotkey.name,
                vscode.SymbolKind.Event,
                null,
                new vscode.Location(
                    hotkey.document.uri,
                    new vscode.Position(hotkey.line, hotkey.character),
                ),
            )));
```

Reply:
This suggestion needs to be apply to whole `symbolProvider.ts` file to keep one style for all code. Why change all legacy working code?

Reply:
I'm not overly concerned with consistency when it comes to using `map` vs `for`, as both are readable enough. However, I prefer `map` in cases like this because we can see the code is shorter -- we've removed noisy boilerplate so we can focus on the task at hand.

We don't have to change every `for` to a `map` in order to get the benefits here. Do you have any concerns making the change for just this loop for now?

Reply:
Done.